control_panel_api/elasticsearch.py

```diff
@@ -0,0 +1,26 @@
+from elasticsearch import Elasticsearch
+from elasticsearch_dsl import Q, Search
+from elasticsearch_dsl.query import Range
+
+
+class ElasticSearch(object):
+    def __init__(self, connection):
+        self.client = Elasticsearch(connection)
+
+    def bucket_hits_aggregation(self, index, bucket_name, day_range=None):
+        s = Search(using=self.client, index=index)
+
+        q1 = Q('term', **{'bucket.keyword': bucket_name})
+        q2 = Q('terms',
```

Comment:
Are these the only operations logged at the moment? Are we interested in other operations? (You probably already asked the same question to Robin)

Reply:
👍